.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -0,0 +1,29 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+jobs:
+- job: osx
+  pool:
+    vmImage: macOS-10.14
```

Comment:
Can you say just `macOS-10`?

Reply:
All files outside of `recipe/` are auto-generated/-updated.